scalacheck/test/src/peschke/UnitSpec.scala

```diff
@@ -6,23 +6,16 @@ import org.scalatest.OptionValues
 import org.scalatest.matchers.must.Matchers
```

Comment:
Should you call this `PropSpec.scala`? Instead of `UnitSpec.scala`?

Reply:
Fair point. `CommonSpec` isn't used either can can be deleted, which was probably why I missed that the file needed to be renamed.